shared/common-adapters/input.native.js

```diff
@@ -1,13 +1,11 @@
-/* @flow */
-
-import React, {Component} from 'react'
-import {TextInput} from 'react-native'
-import {globalColors, globalStyles} from '../styles/style-guide'
+// @flow
+import Box from './box'
 import Platform, {OS} from '../constants/platform'
+import React, {Component} from 'react'
 import Text from './text.native'
-import Box from './box'
-
 import type {Props} from './input'
+import {TextInput} from 'react-native'
+import {globalColors, globalStyles} from '../styles/style-guide'
```

Comment:
Not ideal but simplest without the help of eslint fix

On Friday, July 29, 2016, Chris Nojima <cnojima@keyba.se> wrote:

> Literally sorting the lines
>
> On Friday, July 29, 2016, Marco Munizaga <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In shared/common-adapters/input.native.js
>> <https://github.com/keybase/client/pull/3657#discussion_r72865666>:
>>
>> >  import type {Props} from './input'
>> > +import {TextInput} from 'react-native'
>> > +import {globalColors, globalStyles} from '../styles/style-guide'
>>
>> alphabetized by filename?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/keybase/client/pull/3657/files/919041dd2074c461532c83dc62bd29b4a31876fb#r72865666>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AAfhU0vLhE6InsM4Fs8mzpZtH0tjScTWks5qanTTgaJpZM4JX0TM>
>> .
>>
>